src/Events/Controllers/EventsController.cs

```diff
@@ -20,16 +28,23 @@ namespace Altinn.Platform.Events.Controllers
     [ApiController]
     public class EventsController : ControllerBase
     {
-        private readonly IEventsService _events;
+        private readonly IEventsService _eventsService;
+        private readonly ILogger _logger;
         private readonly GeneralSettings _settings;
+        private readonly string _eventsBaseUri;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="EventsController"/> class.
         /// </summary>
-        public EventsController(IEventsService events, IOptions<GeneralSettings> settings)
+        public EventsController(
+            IEventsService events,
+            ILogger<EventsController> logger,
+            IOptions<GeneralSettings> settings)
         {
-            _events = events;
+            _eventsService = events;
+            _logger = logger;
             _settings = settings.Value;
+            _eventsBaseUri = $"https://platform.{settings.Value.Hostname}";
```

Comment:
this is now set both for app and eventscontroller. How about generating a seperate config value for this baseUri ?

Reply:
I pulled out the whole uri and then we don't need the `Hostname` setting. Should set this with a known good value for our environments.